org.alloytools.util/src/main/java/org/allotools/conversion/util/DTOs.java

```diff
@@ -0,0 +1,114 @@
+package org.allotools.conversion.util;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.lang.reflect.Type;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import aQute.lib.converter.Converter;
+
+public class DTOs {
+
+    @SuppressWarnings("unchecked" )
+    public static <T> T copy(T from, T to) {
+
+        Class< ? extends Object> class1 = from.getClass();
+        assert class1 == to.getClass();
+
+        if (to == null) {
+            try {
+                to = (T) class1.newInstance();
+            } catch (Exception e) {
+                // TODO Auto-generated catch block
+                e.printStackTrace();
+            }
+        }
+
+        for (java.lang.reflect.Field toField : to.getClass().getFields()) {
+            try {
+                java.lang.reflect.Field fromField = class1.getField(toField.getName());
+                Object fromValue = fromField.get(from);
+                Object toValue = convert(toField.getGenericType(), fromValue);
+                toField.set(to, toValue);
+            } catch (Exception e) {
+                //
+            }
+        }
+        return to;
+    }
+
+    @SuppressWarnings("unchecked" )
+    public static <T> T convert(Type type, Object source) {
+        try {
+            Object cnv = Converter.cnv(type, source);
+            return (T) cnv;
+        } catch (Exception e) {
+            // TODO
+            e.printStackTrace();
```

Comment:
Let's not use this anti-pattern of printing the stack trace and re-throwing the exceptoin